### DISCUSSION TITLE
APIKey scheme default name

### DISCUSSION THREAD

**Discussion by BilalAlpaslan** [fastapi/fastapi#9390] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, Security
from fastapi.security import api_key

app = FastAPI()

api_key_header_a = api_key.APIKeyHeader(name='key_a')
api_key_header_b = api_key.APIKeyHeader(name='key_b')


@app.get("/a")
async def a(auth_token: str = Security(api_key_header_a)):
    pass


@app.get('/b')
async def b(auth_token: str = Security(api_key_header_b)):
    pass
```

### Description

openapi.json only contains one of two apikeys

maybe this pr solve this problem: https://github.com/tiangolo/fastapi/pull/4791

### Operating System

Linux, Windows

### FastAPI Version

91.0

### Python Version

3.8

**Comment by YuriiMotov** [comment]
Duplicate of https://github.com/fastapi/fastapi/discussions/6335
